src/dialog-editor/components/field/field.html

```diff
@@ -24,17 +24,41 @@
            type="checkbox">
 
     <!-- date control -->
-    <input ng-switch-when="DialogFieldDateControl"
-           pf-datepicker options="vm.fieldData.options"
-           ng-disabled="true"
-           date="vm.fieldData.default_value">
-
+    <p class="input-group" ng-switch-when="DialogFieldDateControl">
+      <input uib-datepicker-popup="MM/dd/yyyy"
+             type="text"
+             class="form-control"
+             ng-model="vm.fieldData.default_value"
+             is-open="open"
+             datepicker-options="vm.fieldData.options"
+             close-text="{{'Close'|translate}}"/>
+      <span class="input-group-btn">
+        <button type="button"
+                class="btn btn-default"
+                ng-click="open = !open">
+          <i class="fa fa-calendar"></i>
+        </button>
+      </span>
+    </p>
     <!-- date time control -->
     <div ng-switch-when="DialogFieldDateTimeControl">
       <div class="col-sm-6 dateTimePadding">
-        <input ng-model="vm.fieldData.default_value"
-               pf-datepicker options="vm.fieldData.dateOptions"
```

Comment:
`dateOptions` is intentionally removed? (I'm not seeing it defined anywhere, so sounds about right, just checking :), line 33 does have `datepicker-options`, line 53 doesn't)

Reply:
> dateOptions is intentionally removed?

yes.

> line 33 does have datepicker-options

Thanks! Removing :+1: